jenkins/bootstrap.py

```diff
@@ -657,9 +657,10 @@ def pr_paths(base, repos, job, build):
     # Batch merges are those with more than one PR specified.
     pr_nums = pull_numbers(pull)
     if len(pr_nums) > 1:
-        pull = os.path.join(prefix, 'batch')
+        os.environ[PULL_ENV] = 'batch'
```

Comment:
don't change bootstrap.py -- PULL_NUMBER is set by Prow.

Reply:
Done - PTAL.